packages/wheelhouse/src/developmentActions.js

```diff
@@ -47,3 +47,23 @@ export const developmentLog = (pkgName, text) => dispatch => {
 export const developmentBuild = () => async (dispatch, getState) => {
   await dispatch(configLoad());
 };
+
+export const developmentLint = () => async (dispatch, getState) => {
+  await dispatch(configLoad());
+
+  const { packages } = getState();
+  let errored = "";
```

Comment:
I'd do this with an array and `errored.push`... I think as it is right now it will squish all the erroring packages together